packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -355,6 +363,24 @@ describeNoCompat("Named root data stores", (getTestObjectProvider) => {
 
         it("Assign multiple data stores to the same alias, first write wins, " +
         "different containers from snapshot", async () => {
+
+            await setupContainers({
```

Comment:
@andre4i fyi. moving the summarizer setup to just this test, as it makes error inconsistent in the other tests